File: pkg/commands/process/settings/rules/ruby/third_parties/algolia/testdata/datatype_in_save_object.rb
client = Algolia::Search::Client.create('YourApplicationID', 'YourWriteAPIKey')
index = client.init_index("my_index")

index.save_object({ email: user.email }, { auto_generate_object_id_if_not_exist: true })

index.save_objects([{ email: user.email }], { auto_generate_object_id_if_not_exist: true })

File: pkg/commands/process/settings/rules/javascript/third_parties/algolia/testdata/datatype_in_save_object.js
const algoliasearch = require("algoliasearch")
const myAlgolia = algoliasearch("123", "123")

const index = myAlgolia.initIndex("test_index")

// saveObject
const userObj = { user_id: user.ip_address }
index
  .saveObject(userObj, { autoGenerateObjectIDIfNotExist: true })
  .then(console.log("obj saved"))

index.saveObjects([{ email: user.email }])




